middlewares/JWT.go

```
@@ -1,51 +1,60 @@
-	}
-
-	token, err := utils.ValidateJWT(tokenString)
-	if err != nil {
-		utils.LogMessage(utils.LevelError, "Invalid token")
-		utils.LogLineKeyValue(utils.LevelError, "Error", err)
-		utils.LogFooter()
-		return c.Status(fiber.StatusUnauthorized).JSON(fiber.Map{"error": "Invalid token"})
-	}
+		// Update last_activity at most once per day to avoid a write on every request
+		go func() {
+			db.Exec(`UPDATE newf SET last_activity = NOW() WHERE email = $1 AND (last_activity IS NULL OR last_activity < NOW() - INTERVAL '1 day')`, email)
```

golangci-lint (2.13.2)

[error] Error return value of `db.Exec` is not checked

(errcheck)